### ISSUE TITLE
Kind test script Istio install failing with kubectl version v1.18.x

### ISSUE THREAD

**Issue by glindsell** [SeldonIO/seldon-core#1802]
testing/scripts/kind_test_setup.sh fails with following errors due to CRD not being present:

```
unable to recognize "istio-1.4.2.yaml": no matches for kind "Gateway" in version "networking.istio.io/v1alpha3"
unable to recognize "istio-1.4.2.yaml": no matches for kind "VirtualService" in version "networking.istio.io/v1alpha3"
unable to recognize "istio-1.4.2.yaml": no matches for kind "EnvoyFilter" in version "networking.istio.io/v1alpha3"
unable to recognize "istio-1.4.2.yaml": no matches for kind "DestinationRule" in version "networking.istio.io/v1alpha3"
unable to recognize "istio-1.4.2.yaml": no matches for kind "Policy" in version "authentication.istio.io/v1alpha1"
unable to recognize "istio-1.4.2.yaml": no matches for kind "Gateway" in version "networking.istio.io/v1alpha3"
unable to recognize "istio-1.4.2.yaml": no matches for kind "Sidecar" in version "networking.istio.io/v1alpha3"
unable to recognize "istio-1.4.2.yaml": no matches for kind "MeshPolicy" in version "authentication.istio.io/v1alpha1"
unable to recognize "istio-1.4.2.yaml": no matches for kind "DestinationRule" in version "networking.istio.io/v1alpha3"
unable to recognize "istio-1.4.2.yaml": no matches for kind "attributemanifest" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "attributemanifest" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "handler" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "instance" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "instance" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "rule" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "rule" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "instance" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "instance" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "instance" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "instance" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "instance" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "instance" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "instance" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "instance" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "handler" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "rule" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "rule" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "rule" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "rule" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "handler" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "rule" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "rule" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "instance" in version "config.istio.io/v1alpha2"
unable to recognize "istio-1.4.2.yaml": no matches for kind "DestinationRule" in version "networking.istio.io/v1alpha3"
```

This has only been found to happen when using the following version of kubectl client:
```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.2", GitCommit:"52c56ce7a8272c798dbc29846288d7cd9fbae032", GitTreeState:"clean", BuildDate:"2020-04-21T01:25:41Z", GoVersion:"go1.13.10", Compiler:"gc", Platform:"linux/amd64"}
```

Versions v1.17.x and v1.16.x of kubectl client do not fail.

Additionally if using v1.18.x, if the CRDs are installed separately as shown [here](https://istio.io/docs/setup/install/helm/#installation-steps) before Istio is installed, then the errors do not occur.

**Comment by glindsell** [comment]
Related issues:
https://github.com/kubernetes-sigs/kind/issues/762
https://github.com/istio/istio/issues/11409
https://github.com/knative/serving/issues/2195

**Comment by glindsell** [comment]
#1808 pins the test scripts to kubectl v1.17.0

**Comment by glindsell** [comment]
Fixed by #1810
